class/nfs-client-provisioner.yml

```
@@ -1,11 +1,32 @@
 parameters:
   kapitan:
+    dependencies:
+      - type: https
```

We can use the native helm dependency type

```suggestion
      - type: helm
```